tests/__init__.py

```diff
@@ -90,7 +91,7 @@ def newfunc (*args, **kwargs):
             if not testfunc():
                 pytest.skip("%s is not available" % name)
             return func(*args, **kwargs)
-        newfunc.func_name = func.func_name
+        newfunc.__name = func.__name__
```

Comment:
Typo: `__name` should be `__name__`.

In fact I suggest using `@functools.wraps(func)` to decorate the `def newfunc` instead of copying metadata by hand. It'll also take care to copy `__doc__` and `__module__` and all sorts of other important things.

Reply:
Good to know. Added `wraps`.